src/services/state-persistor/utils/utils.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module 'ngeo.utils' {
+  /**
+   * Checks if on Safari.
+   * @return {boolean} True if on Safari.
+   */
+  export function isSafari(): boolean
```

Comment:
all this looks like utilities function, better be moved outside` state-persistor` folder

Reply:
removed utils file (only rgb function used)